egui/src/widgets/plot/items.rs

```diff
@@ -273,7 +274,8 @@ impl Line {
 
     /// Name of this line.
     ///
-    /// This name will show up in the plot legend, if legends are turned on.
+    /// This name will show up in the plot legend, if legends are turned on. Multiple lines may
+    /// share the same name.
```

Comment:
But if two items share the same name, they also share the same legend entry, correct?

If so we should document that here and below

Reply:
Added the comment.